apps/system/style/chrome/chrome.css

```diff
@@ -386,6 +392,8 @@
 .controls .back-button[disabled],
 .controls .forward-button[disabled] {
   display: none;
+  opacity: 0;
+  animation: fadeOut .2s ease-in;
```

Comment:
I'm still not sure how to transition opacity if we want to add display:none at the end (we need that, because the input element is relative).

https://jsfiddle.net/30hocep7/

Am I missing something?